### PR TITLE
[ADD] estate: Add demo y data files

### PR DESCRIPTION
**Pull request by gerashdo**
**Define module data**
- Create `estate.property.type.csv` to load initial property type data.
- Create `estate.property` demo data.
- Create `estate.property.offer` demo data that includes relationships, and functions to accept or refuse the offers.
- Create `estate.property` demo data with inline related offers.
- [Reference link](https://www.odoo.com/documentation/16.0/es/developer/tutorials/define_module_data.html#)

### CODE REVIEW

**Comment by gerashdo** [comment]
@Gilmaciasr Ready for review.

File: estate/models/estate_property.py [hunk 1]
```diff
@@ -46,7 +46,8 @@ class EstateProperty(models.Model):
     )
     property_type_id = fields.Many2one("estate.property.type")
     buyer_id = fields.Many2one("res.partner", copy=False)
-    salesperson_id = fields.Many2one("res.users", string="Salesman", default=lambda self: self.env.user)
+    # salesperson_id = fields.Many2one("res.users", string="Salesman", default=lambda self: self.env.user)
```
Comment (Gilmaciasr) [inline estate/models/estate_property.py]: Delete commented lines.
Also, why did you remove the default salesperson?
On chapter 8 it requests to set the current user as salesperson default 
image
https://www.odoo.com/documentation/16.0/developer/tutorials/getting_started/08_relations.html

**Comment by gerashdo** [comment]
@Gilmaciasr Ready for review 😅.
